cgroup/README.md

```diff
@@ -0,0 +1,48 @@
+# Cgroup Check
+
+## Overview
+
+The cgroup check lets you:
+
+* Collect resource usage metrics for specific linux control groups (cgroups): CPU, memory, I/O, etc
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host.
+
+### Installation
+
+The cgroup check is included in the [Datadog Agent][1] package, so you don't need to install anything else on your server.
+
+### Configuration
+
+The cgroup check doesn't monitor anything by default; you must tell it which processes you want to monitor, and how.
```

Comment:
Can we specify more details about how to configure this? Assuming it's the same as configuration for the process check, but let's explicitly say "edit the conf.yaml" if that's the case.